integr8ly/tutorial-web-app
# APPDUX-228: Fixes to incorporate latest PF4 breaking change release

## Motivation
https://issues.redhat.com/browse/APPDUX-228

## What
Update to the latest Patternfly 4 build and make fixes so the UI behavior and look and feel work as expected.

## Why
To incorporate the latest PF4 components going forward.

## Verification Steps
Open Solution Explorer and traverse the entire UI to verify that no new issues were introduced.

## Checklist:

## Progress

## Additional Notes
You can test effectively by pulling down this branch and running locally, but if you want to test on a server, you can use this docker image of the solution explorer:

docker.io/mfrances17/dev-tutorial-web-app:pf4-latest

**Various screen caps:**
![UI-pf4-1](https://user-images.githubusercontent.com/39063664/85892141-638bd880-b7be-11ea-9c9d-8d5da37c6a7a.png)

![UI-pf4-2](https://user-images.githubusercontent.com/39063664/85892164-6dadd700-b7be-11ea-9c00-c3af3c661fb9.png)

![UI-pf4-3](https://user-images.githubusercontent.com/39063664/85892171-70a8c780-b7be-11ea-976d-e2b812d796a1.png)

![UI-pf4-4](https://user-images.githubusercontent.com/39063664/85892178-73a3b800-b7be-11ea-9a9b-9190817f913f.png)

![UI-pf4-5](https://user-images.githubusercontent.com/39063664/85892182-76061200-b7be-11ea-84da-917240fad132.png)

![UI-pf4-6](https://user-images.githubusercontent.com/39063664/85892187-78686c00-b7be-11ea-9dae-c841bd986b48.png)
